seed.js

```diff
@@ -0,0 +1,90 @@
+const { users, categories, thread, comments } = require('./models')
+
+const main = async () => {
+    
+    
+
+    // Create users
+    await users.create({
+        id: 1,
+        name: 'Imani Paul',
+        email: 'imani@email.com',
+        role: 'student',
+        password_digest: 'password',        
+    })
+
+    await users.create({
+        id: 2,
+        name: 'Alane Reyes',
+        email: 'alane@email.com',
+        role: 'student',
+        password_digest: 'password',        
+    })
+
+    await users.create({
+        id: 3,
+        name: 'Soren Soroush',
+        email: 'soren@email.com',
+        role: 'student',
+        password_digest: 'password',        
+    })
+
+    await users.create({
+        id: 4,
+        name: 'Solito Reyes',
+        email: 'solitonyc@gmail.com',
+        role: 'student',
+        password_digest: 'password',        
+    })
+
+    // Create categories
+    await categories.create({
+        id: '1',
+        name: 'homework',        
+    })
+
+    await categories.create({
+        id: '2',
+        name: 'projects',        
+    })
+
+    await categories.create({
+        id: '3',
+        name: 'lesson',        
+    })
+
+    await categories.create({
+        id: '4',
+        name: 'services',        
+    })
+
+    await categories.create({
+        id: '5',
+        name: 'food',        
+    })
+
+    await categories.create({
+        id: '6',
+        name: 'student_life',        
+    })
+
+    //Create thread
+    await thread.create({
+        id: 1,        
+        category_id: 6,
```

Comment:
For relating the threads to categories, I think we need to do a set function. for example thread1.setCategory(Student_life)

Reply:
Ok implemented for next push.